### ISSUE TITLE
Fabric Roadmap

### ISSUE THREAD

**Issue by igenroot** [sarriaroman/FabricPlugin#147]
From fabric website, stated that Fabric Dashboard can be access until mid 2019.  So if that happen, still we can use fabric plugin for Crashlytics on firebase?

**Comment by amymarsh4work** [comment]
Yes. We have setup Crashlytics on Firebase and it is working well. My understanding is that eventually the Fabric Dashboard will go away, but at that point, you should be migrated to Firebase and be viewing your crash data over there instead. Follow the migration instructions and you will be fine.
